c/nla-digbench/LICSENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
The license file could be replaced by a symlink to the Apache license in the root directory.